src/module/actor/data-model-character.test.js

```diff
@@ -1,60 +1,47 @@
-import OseDataModelCharacter from "./data-model-character"
+/**
+ * @file Quench unit tests for the Character data model class.
+ */
+export const key = "ose.datamodel.character";
+export const options = { displayName: "Character Data Model" };
 
-export const key = 'ose.datamodel.character';
-export const options = { displayName: 'Character Data Model'}
+// @todo How can we add items to this actor?
+const createMockCharacter = (system = {}, items = [], characterOptions = {}) =>
```

Comment:
does this actually belong outside of our export or was this an auto-fix we don't like?

Reply:
This came from the linter; it wanted arrow functions that don't rely on context from their block to be lifted to a higher scope.